terraform/deployments/apps/test/content-store/variables.tf

```diff
@@ -0,0 +1,5 @@
+variable "image_tag" {
```

Comment:
Is there a reason for outputs.tf being symlinked but not variables.tf? If I understand the design correctly, these modules all have the same interface and that's required in order for Concourse to be able to use them(?)

Same question for modules/task-definitions/router/outputs.tf

Reply:
Good spot, thanks. I'll address this in a subsequent PR since I'm going to look at removing the `common.tf` and `output.tf` files as per @richardTowers's suggestion below.